### DISCUSSION TITLE
[Simulation question]: How to score the lesion time in SDD file for nucleus scorer example

### DISCUSSION THREAD

**Discussion by Geoffvinc** [topas-nbio/TOPAS-nBio#29] [Simulation-question]
### Simulation question

Hi everyone, I tried to simulate DNA repair from the sdd output of the nucleus scorer example (examples/scorers/SBSDamageToDNANucleus) using DaMaRiS. However, I encountered the error below:
> ERROR: STDInput not successfully read, event vector not populated
> WARNING - Attempt to delete the physical volume store while geometry closed !
> WARNING - Attempt to delete the logical volume store while geometry closed !
> WARNING - Attempt to delete the solid store while geometry closed !
> WARNING - Attempt to delete the region store while geometry closed !

To troubleshoot this, I compared the sdd output of the nucleus scorer example and the sdd output of the DaMaRiS example.
The sdd output of DaMaRiS example contains
> Data entries, 1, 1, 1, 1, 1, 1, 0, 0, 1, 0, 0, 0, 0, 0;

which 9th field encodes for the lesion time, according to the [SDD publication.
](https://bioone.org/journals/radiation-research/volume-191/issue-1/RR15209.1/A-New-Standard-DNA-Damage-SDD-Data-Format/10.1667/RR15209.1.full)
Meanwhile, the sdd output of the nucleus scorer example does not contain the lesion time. as can be seen below
> Data entries, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0;

Therefore, I assume the error is caused because the DaMaRiS requires the lesion time data to run.
This observation is validated when I deleted the lesion time of the DaMaRiS example, and it returned the same error.

Does anyone know how to score the lesion time for nucleus scorer example or whether if there is any alternative solution to run the DaMaRiS using the nucleus scorer example?

Thank you for your time and attention.

Regards,
Geoff

### TOPAS-nBio category

Scoring

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by nknanfeng** [comment]
Sorry, I noticed that you are also running examples related to Nucleus. Can you run the example /examples/geometry/nucleusModel/Nucleus.txt normally? If so, can you tell me about the computer hardware configuration you are using? I want to know if it is because my personal computer's hardware performance is insufficient that running this example cannot obtain ideal results.
I would be very grateful for your reply.
